WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -25,12 +29,20 @@ public partial class AdditionalInformation : Window
         private List<TextBox> textBoxes = new List<TextBox>();
         private TextBox name = new TextBox();
         private Grid DynamicGrid = new Grid();
+        private int floor;
```

Comment:
Why a hospital map should have floor data?

Reply:
It is in AdditionInformation, and it is used for refreshing map after update.